client/src/components/Pages/login/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import {
+  NotificationContainer,
+  NotificationManager,
+} from 'react-notifications';
+
+import loginSchema from '../../utils/validationSchemas/loginSchema';
+import LabeldInput from '../../common/LabeledInput';
+import Button from '../../common/Button';
+import 'react-notifications/lib/notifications.css';
+import './style.css';
+
+class Login extends Component {
+  state = {
+    account: {
+      email: '',
+      password: '',
+    },
+    error: {},
+  };
+
+  handleSubmit = async event => {
+    event.preventDefault();
+    const { account: loginAccount } = this.state;
+    try {
+      await loginSchema.validate(loginAccount, {
+        abortEarly: false,
+      });
+      await axios.post('/api/v1/login', loginAccount);
```

Comment:
Show notification for the user with the message from the back end 
check the route /login or see the response